docs/tutorials/quickstart.md

```diff
@@ -1,186 +1,240 @@
 ```
 
-## Building on Windows, OSX, and Linux
-
-Building AKS Engine from source has a few requirements for each of the platforms. Download and install the pre-reqs for your platform, Windows, Linux, or Mac:
-
-### Prerequisite
-
-1. Go version 1.8 [installation instructions](https://golang.org/doc/install)
-2. Git Version Control [installation instructions](https://git-scm.com/download/)
-
-### Windows
-
-Setup steps:
-
-* Setup your go workspace. This guide assumes you are using `c:\gopath` as your Go workspace:
-  1. Type Windows key-R to open the run prompt
-  2. Type `rundll32 sysdm.cpl,EditEnvironmentVariables` to open the system variables
-  3. Add `c:\go\bin` and `c:\gopath\bin` to your PATH variables
-  4. Click "new" and add new environment variable named `GOPATH` and set the value to `c:\gopath`
-
-* Build aks-engine:
-  1. Type Windows key-R to open the run prompt
-  2. Type `cmd` to open a command prompt
-  3. Type `mkdir %GOPATH%` to create your gopath
-  4. Type `cd %GOPATH%`
-  5. Type `go get -d github.com/Azure/aks-engine` to download aks-engine from GitHub
-  6. Type `go get all` to get the supporting components
-  7. Type `go get -u github.com/go-bindata/go-bindata/...`
-  8. Type `cd %GOPATH%\src\github.com\Azure\aks-engine\pkg\engine`
-  9. Type `go generate`
-  10. Type `cd %GOPATH%\src\github.com\Azure\aks-engine\pkg\i18n`
-  11. Type `go generate`
-  12. Type `cd %GOPATH%\src\github.com\Azure\aks-engine`
-  13. Type `go build` to build the project
-  14. Type `go install` to install the project
-  15. Run `aks-engine.exe` to see the command line parameters
-
-### OS X and Linux
-
-Setup steps:
-
-  1. Open a command prompt to setup your gopath:
-  2. `mkdir $HOME/go`
-  3. Edit `$HOME/.bash_profile` and add the following lines to setup your go path:
-      ```sh
-      export GOPATH=$HOME/go
-      export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
-      ```
-  4. `source $HOME/.bash_profile`
-
-Build aks-engine:
-
-  1. Type `go get github.com/Azure/aks-engine` to get the aks-engine Github project
-  2. Type `cd $GOPATH/src/github.com/Azure/aks-engine` to change to the source directory
-  3. Type `make bootstrap` to install supporting components
-  4. Type `make build` to build the project
-  5. Type `./bin/aks-engine` to see the command line parameters
+* To enable the optional network policy enforcement using calico, you have to set the parameter during this step according to this [guide](../topics/features.md#optional-enable-network-policy-enforcement-using-calico)
+* To enable the optional network policy enforcement using cilium, you have to set the parameter during this step according to this [guide](../topics/features.md#optional-enable-network-policy-enforcement-using-cilium)
+
+Now we can deploy the files `azuredeploy.json` and `azuredeploy.parameters.json` using either the Azure CLI or PowerShell.
+
+Using the CLI:
+
+```text
```

Comment:
```suggestion
```sh
```

Reply:
switched to `console` to match the rest of the docs. Thanks!